src/model/cloud-runner/workflows/build-automation-workflow.ts

```
@@ -50,65 +50,141 @@ export class BuildAutomationWorkflow implements WorkflowInterface {
-    )} && git clone -q -b ${CloudRunner.buildParameters.cloudRunnerBranch} ${
-      CloudRunnerFolders.unityBuilderRepoUrl
-    } "${CloudRunnerFolders.ToLinuxFolder(CloudRunnerFolders.builderPathAbsolute)}" && chmod +x ${builderPath}`;
-
-    const cloneBuilderCommands = `if [ -e "${CloudRunnerFolders.ToLinuxFolder(
-      CloudRunnerFolders.uniqueCloudRunnerJobFolderAbsolute,
-    )}" ] && [ -e "${CloudRunnerFolders.ToLinuxFolder(
-      path.join(CloudRunnerFolders.builderPathAbsolute, `.git`),
-    )}" ] ; then echo "Builder Already Exists!" && tree ${
-      CloudRunnerFolders.builderPathAbsolute
-    }; else ${commands} ; fi`;
+    )}
```

ESLint

[error] Replace `⏎······CloudRunnerFolders.builderPathAbsolute,⏎····` with `CloudRunnerFolders.builderPathAbsolute`

(prettier/prettier)